### PR TITLE
Addition of contracts

### PR DESCRIPTION
**Pull request by Aurige**
Creation of of a contract object following Entur's following the Basecamp discussion (https://3.basecamp.com/3256016/buckets/2570434/messages/5757225499 ) and the meeting discussion/decision. 
contract can be standalone (in Frame) or attached to a Role Assignment
May require discussion with the Transmodel group

### CODE REVIEW

**Comment by Aurige** [comment]
This is an initial draft proposal for discussion with the group on meeting # 15

File: xsd/netex_framework/netex_responsibility/netex_responsibilitySet_version.xsd [hunk 1]
```diff
@@ -347,5 +349,127 @@ Rail transport, Roads and Road transport
...
+						<xsd:sequence>
+							<xsd:group ref="EntityInVersionGroup" minOccurs="0"/>
+						</xsd:sequence>
+						<xsd:sequence>
+							<xsd:group ref="DataManagedObjectGroup"/>
+						</xsd:sequence>
+						<xsd:sequence>
+							<xsd:group ref="ContractGroup">
+								<xsd:annotation>
+									<xsd:documentation>Details for a Contract</xsd:documentation>
```
Comment (skinkie) [inline xsd/netex_framework/netex_responsibility/netex_responsibilitySet_version.xsd:370]:
```suggestion
									<xsd:documentation>Details for a Contract.</xsd:documentation>
```

File: xsd/netex_framework/netex_responsibility/netex_responsibilitySet_version.xsd [hunk 2]
```diff
@@ -347,5 +349,127 @@ Rail transport, Roads and Road transport
...
+								</xsd:annotation>
+							</xsd:group>
+						</xsd:sequence>
+					</xsd:sequence>
+					<xsd:attribute name="id" type="ContractIdType"/>
```
Comment (skinkie) [inline xsd/netex_framework/netex_responsibility/netex_responsibilitySet_version.xsd:375]:
```suggestion
					<xsd:attribute name="id" type="ContractIdType" use="required"/>
```

File: xsd/netex_framework/netex_responsibility/netex_responsibilitySet_version.xsd [hunk 3]
```diff
@@ -347,5 +349,127 @@ Rail transport, Roads and Road transport
...
+					<xsd:documentation>binding  nonBinding</xsd:documentation>
+				</xsd:annotation>
+			</xsd:element>
+			<xsd:element name="ContractGoverningLaw" type="MultilingualString" minOccurs="0"/>
+			<xsd:element name="contractors">
+				<xsd:annotation>
+					<xsd:documentation>The contractor is the party which agrees to a written contract.</xsd:documentation>
+				</xsd:annotation>
+				<xsd:complexType>
+					<xsd:sequence minOccurs="0" maxOccurs="1">
```
Comment (skinkie) [inline xsd/netex_framework/netex_responsibility/netex_responsibilitySet_version.xsd:427]: Curious if this works :-)

**Comment by Aurige** [comment]
Thanks @skinkie ... I will update de PR accodingly

File: xsd/netex_framework/netex_responsibility/netex_responsibilitySet_version.xsd [hunk 4]
```diff
@@ -347,5 +349,132 @@ Rail transport, Roads and Road transport
...
+			</xsd:element>
+			<xsd:element name="LegalStatus" type="LegalStatusEnumeration" minOccurs="0">
+				<xsd:annotation>
+					<xsd:documentation>binding  nonBinding</xsd:documentation>
+				</xsd:annotation>
+			</xsd:element>
+			<xsd:element name="ContractGoverningLaw" type="MultilingualString" minOccurs="0"/>
+			<xsd:element name="contractors">
+				<xsd:annotation>
+					<xsd:documentation>The contractor is that party the provides agreed services or goods to the Contractee according to the conditions in a Contract</xsd:documentation>
```
Comment (skinkie) [inline xsd/netex_framework/netex_responsibility/netex_responsibilitySet_version.xsd:424]:
```suggestion
					<xsd:documentation>The contractor is that party the provides agreed services or goods to the Contractee according to the conditions in a Contract.</xsd:documentation>
```

**Review by skinkie — APPROVED** [review]

**Review by ue71603 — APPROVED** [review]

**Review by JohanEntur — APPROVED** [review]

**Comment by skinkie** [comment]
@JohanEntur can you make a pull request with a single example of a contract?

**Comment by JohanEntur** [comment]
A team in Entur will begin using contracts now.

**Pull request merged**